### ISSUE TITLE
Parse errors when last element in structure has trailing comma

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#263]
When a structure is created shorthand, where the last element has a trailing comma:

```cf
foo = {
  bar = 1,
  baz = 2,
};
```

(which seems to work fine in Railo and is commonly supported by other languages), this triggers a number of parsing errors:

```xml
<issue severity="ERROR" id="PARSE_ERROR" message="PARSE_ERROR" category="CFLint" abbrev="PE">
  <location file="C:\Inbox\cflint-test\.\test.cfc" fileName="test.cfc" function="" column="2" line="7" message="Unable to parse" variable="">
    <Expression><![CDATA[no viable alternative at input '{\r\n\t\t    bar=1,\r\n\t\t    baz=2,\r\n\t\t}']]></Expression>
  </location>
</issue>
<issue severity="ERROR" id="MISSING_SEMI" message="MISSING_SEMI" category="CFLint" abbrev="MS">
  <location file="C:\Inbox\cflint-test\.\test.cfc" fileName="test.cfc" function="" column="13" line="5" message="End of statement(;) expected instead of ," variable="">
    <Expression><![CDATA[,]]></Expression>
  </location>
</issue>
<issue severity="ERROR" id="MISSING_SEMI" message="MISSING_SEMI" category="CFLint" abbrev="MS">
  <location file="C:\Inbox\cflint-test\.\test.cfc" fileName="test.cfc" function="" column="13" line="6" message="End of statement(;) expected instead of ," variable="">
    <Expression><![CDATA[,]]></Expression>
  </location>
</issue>
```

The same applies with nested structures. There will be similar errors reported when this is scanned:

```cf
var foo = {
  bar = 1,
  baz = {
    foo = 2,
  }
};
```

**Comment by ryaneberly** [comment]
fixed in cfparser 2.4.5
